docs/nuget-org/nuget-org-faq.yml

```diff
@@ -228,11 +228,17 @@ sections:
           
       - question: How do I change my NuGet.org account username?
         answer: |
-          You cannot. As a matter of policy we do not allow the change of usernames. Also, doing so is a breaking change for users that may have defined [package trust policies based on the package owner](../consume-packages/installing-signed-packages.md#trust-package-owners). The only way to change your username is to create a new account with the desired username. We recommend you delete your existing account before you create a new one, otherwise you will not be able to reuse your registered Microsoft account.
+          You can request a username change by sending an email to [support@nuget.org](mailto:support@nuget.org) from the email address that is attached to the account you want to update.
+          Be sure to include the old username and the new username you would like to change to.
+          We will then review your request and, upon approval, get confirmation from you that we are about the take the correct action and that you understand the consequences.
```

Comment:
Can we call out what the consequences are? i.e. one way? etc.

Reply:
Added a new line under important about not being able to revert the change. I think the important section covers the rest of the potential consequences.